lib/classes/ConfigSchemaHandler/resolveAjvValidate.js

```diff
@@ -17,8 +17,13 @@ const getCacheDir = () => {
   );
 };
 
+const cachedValidatorsBySchemaHash = {};
+
 const getValidate = async (schema) => {
   const schemaHash = objectHash(deepSortObjectByKey(schema));
+  if (cachedValidatorsBySchemaHash[schemaHash]) {
```

Comment:
It'll be slightly cleaner to rely on `memoizee`, so do something as:

```javascript
const getValidateForSchemaHash = memorize(hash => { ... }):

module.exports = schema => getValidateForSchemaHash(objectHash(deepSortObjectByKey(schema))
```

Reply:
I was thinking initially about using memoize, but honestly, it's becoming less readable for me if memoizee is used. To the memoized function, you still need to pass the original `schema` arg because if it wasn't cached, it needs to be generated first based on `schema`. Do you think using `memoize` here actually would improve readability/produce more clean solution?